runtime/v2/module.go

```diff
@@ -206,17 +206,17 @@ func ProvideMemoryStoreKey(key depinject.ModuleKey, app *AppBuilder) *storetypes
 
 func ProvideKVStoreService(config *runtimev2.Module, key depinject.ModuleKey, app *AppBuilder) store.KVStoreService {
 	storeKey := ProvideKVStoreKey(config, key, app)
-	return kvStoreService{key: storeKey}
+	return stf.NewKVStoreService([]byte(storeKey.Name()))
```

Comment:
should use unsafe since the is quite heavy, i think there is a helper function in stf

Reply:
There is, but im not sure if its public